include/pros/adi.hpp

```diff
@@ -190,6 +190,26 @@ class Port {
 	 */
 	std::int32_t set_value(std::int32_t value) const;
 
+	/**
+	 * Gets the port of the sensor.
+	 *
+	 * \return 1 if the operation was successful or PROS_ERR if the operation
+	 * failed, setting errno.
```

Comment:
Can we clarify this return a bit more. It doesn't return 1 or pros error it returns a tuple with 3 values instead.
Can we also specify what is in each position of the tuple and state here what the smartport value for the non-expander is